README.md

```diff
@@ -6,6 +6,10 @@ _Write once, run on any cloud ☁️_
 [![godoc](https://godoc.org/github.com/google/go-cloud?status.svg)][godoc]
 [![Coverage](https://codecov.io/gh/google/go-cloud/branch/master/graph/badge.svg)](https://codecov.io/gh/google/go-cloud)
```

Comment:
We previously had the logo before the first heading (see 11179ad85494f2e152628bcb79bddaba42dab4ca).

Reply:
Yes, I'm aware, but the new location seemed like a more common place to put logos in. For example see the github repos of moby, kubernetes, go, ...

Reply:
OK.